R/deployments.R

```diff
@@ -147,6 +148,7 @@ deploymentRecord <- function(name,
     appId = appId %||% "",
     bundleId = bundleId %||% "",
     url = url %||% "",
+    when = when %||% as.numeric(as.POSIXct(Sys.time())),
```

Comment:
Please add a test confirming that we read the same `when` data that was written.